crates/ruff_linter/resources/test/fixtures/fastapi/FAST003.py

```diff
@@ -144,3 +144,35 @@ async def read_thing(query: str):
 @app.get("/things/{thing_id=}")
 async def read_thing(query: str):
     return {"query": query}
+
+
+# https://github.com/astral-sh/ruff/issues/13657
+def dependable(thing_id): ...
+def not_so_dependable(lorem): ...
```

Comment:
Nit: Can we rename this to `dependency_with_thing_id_param` and `dependency_without_thing_id_param`